dev/io.openliberty.mcp.internal_fat/fat/src/io/openliberty/mcp/internal/fat/tool/ToolTest.java

```diff
@@ -773,10 +883,16 @@ public void testToolList() throws Exception {
                                                 "value"
                                             ]
                                         },
+                                        "annotations": {
+                                          "readOnlyHint": true,
+                                          "destructiveHint": false,
+                                          "idempotentHint": true,
+                                          "openWorldHint": false
+                                        },
```

Comment:
Although all the annotations in this test are correct for the methods that they're annotating, can we also test some different combinations of hints?

Also, can we make it so that we only include the annotations if they're different to the default? If there are no non-default annotations we can omit the annotations object altogether.